spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/integration/ReadIntegrationTestBase.java

```diff
@@ -456,6 +456,30 @@ private void testBigLakeTable(FormatOptions formatOptions, String dataFileName,
     testShakespeare(df);
   }
 
+  @Test
+  public void testReadFromTableSnapshot() {
+    String snapshot = String.format("%s.%s.snapshot", PROJECT_ID, testDataset);
```

Comment:
Please replace to `String.format("%s.%s.%s_snapshot", PROJECT_ID, testDataset, testTable);` in order to avoid constant table name